SwiftLeeds/Views/Sponsors/SponsorsView.swift

```diff
@@ -93,13 +95,35 @@ private extension SponsorsView {
                 }
             }.foregroundColor(.clear)
     }
+    
+    func jobtile(for sponsor: Sponsor) -> some View {
+        ForEach(sponsor.jobs) { job in
+            CommonTileButton(primaryText: job.title,
```

Comment:
minor: I think it's important to include the location as part of the job tile - this is a key part of whether the role is even close to appropriate for the individual.